fp.js

```diff
@@ -6,7 +6,7 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
+people.map(person => {
   peoplesAges.push(person.age)
 })
```

Comment:
Remember that `map`, unlike `forEach`, returns an array. What ends up in that array depends on what you are returning from the function passed into `map`. So it should look something like...

```js
const ages = people.map(person => {
  // for each person, we can to retrieve their age and put it in the new array
  return person.age;
})
```